assets/js/components/IngestSheet/Completed.jsx

```diff
@@ -34,8 +34,8 @@ const IngestSheetCompleted = ({ sheetId }) => {
         30%
       </progress>
     );
-  if (worksError) return <Error error={worksError.message} />;
-  if (errorsError) return <Error error={errorsError.message} />;
```

Comment:
Just making sure this change is intentional?

Reply:
Yes, this was causing errors. Looks like it was modified on of your branches.